docs/tools/mbed_targets.md

```diff
@@ -195,7 +195,29 @@ The `post_build_hook` for the `TEENSY3_1` converts the output file (`binf`) from
 
 Use this property to pass necessary data for exporting to various third party tools and IDEs and for building applications with bootloaders.
 
-Please see <a href="/docs/v5.6/tools/exporting.html" target="_blank">our exporting page</a> for information about this field.
+We use the tool [ArmPackManager](https://github.com/ARMmbed/mbed-os/tree/master/tools/arm_pack_manager) to scrape [MDK5 Software Packs](https://www.keil.com/dd2/Pack/) for target information by parsing [http://www.keil.com/pack/index.idx](http://sadevicepacksprod.blob.core.windows.net/idxfile/index.idx). [index.json](https://github.com/ARMmbed/mbed-os/blob/master/tools/arm_pack_manager/index.json) stores the relevant information from the [PDSC (Pack Description)](http://www.keil.com/pack/doc/CMSIS/Pack/html/) retrieved from each CMSIS PACK described in the index.
+
+A device support`.pdsc` file typically describes a family of devices. Each device is uniquely identified by its [device name](/docs/v5.6/reference/contributing-target.html). This name makes a natural key to associate a device with its information in `index.json`. To support IAR and uVision exports for your target, You must add a [device_name](/docs/v5.6/reference/contributing-target.html) field in `targets.json` containing this key.
```

Comment:
- Query: Can we delete "typically"?
- Please change the second sentence to "The device name uniquely identifies each device."
- Please change the first link to:
```<a href="/docs/v5.6/reference/contributing-target.html" target="_blank">device name</a>```
- Please lower case "you" in the last sentence.
- Query: Why does "device_name" have an underscore here but not two sentences earlier? Are they different? If not, please delete the link because we already linked that word two sentences earlier. If they are different, please use the link from above.

Reply:
We can typically delete "typically" 😄 

It should either be "device name" as in the name for a device or `"device_name"` the key in `targets.json`